### PR TITLE
y2tIi3o3: PaginationHelper task solved

### PR DESCRIPTION
**Pull request by MykytaMusaiev**
task ready

### CODE REVIEW

File: algorithms/PaginationHelper.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    pageItemCount(pageIndex) {
+        let result = this.pages.length - (pageIndex *  this.pageDivider)
+        if(result <=0 || pageIndex < 0) return -1;
+        return (result > this.pageDivider) ? this.pageDivider : result
+    }
+
+    pageIndex(itemIndex) {
+        if (itemIndex < 0 || itemIndex > this.pages.length - 1) {
```
Comment (TaHUoP) [inline algorithms/PaginationHelper.js]: Can be converted to a ternary operator

File: algorithms/PaginationHelper.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// For this exercise you will be strengthening your page-fu mastery.
+// You will complete the PaginationHelper class, which is a utility class helpful for querying paging information related to an array.
+// The class is designed to take in an array of values and an integer indicating how many items will be allowed per each page.
+// The types of values contained within the collection/array are not relevant.
+
+class PaginationHelper {
+    constructor(pages, pageDivider) {
+        this.pages = pages
+        this.pageDivider = pageDivider
```
Comment (TaHUoP) [inline algorithms/PaginationHelper.js]: I suggest renaming to pageSize as more obvious option

File: algorithms/PaginationHelper.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        return Math.ceil(this.pages.length / this.pageDivider)
+    }
+
+    itemCount() {
+        return this.pages.length;
+    }
+
+    pageItemCount(pageIndex) {
+        let result = this.pages.length - (pageIndex *  this.pageDivider)
+        if(result <=0 || pageIndex < 0) return -1;
```
Comment (TaHUoP) [inline algorithms/PaginationHelper.js]: add a space after if and put the return on a separate line

File: algorithms/PaginationHelper.js (hunk 1 above)
Comment (anotherrandomcoder) [reply algorithms/PaginationHelper.js:27]: agreed and fixed

Comment (MykytaMusaiev) [reply algorithms/PaginationHelper.js]: fixed

File: algorithms/PaginationHelper.js (hunk 2 above)
Comment (MykytaMusaiev) [reply algorithms/PaginationHelper.js]: fixed

**Review by TaHUoP — APPROVED** [review]

**Pull request merged**